src/platform/packages/shared/kbn-esql-ast/src/definitions/utils/expressions.ts

```diff
@@ -251,3 +258,60 @@ export function getExpressionType(
 
   return 'unknown';
 }
+
+/**
+ * Returns the left or right operand of a binary expression function.
+ */
+export function getBinaryExpressionOperand(
```

Comment:
This isn't strictly necessary, but I'm always confused about what to look for in args. Perhaps some small functions that help navigate these complex structures would help, like lodash or rambajs.